phenol-core/src/main/java/org/monarchinitiative/phenol/ontology/data/TermSynonym.java

```diff
@@ -67,4 +151,33 @@ public String toString() {
         + termXrefs
         + "]";
   }
+
+
+  public boolean hasSynonymType() {
```

Comment:
Why no also have a isExact() or isRelated() method to prevent having to expose the TermSynonymScope more than necessary?